### PR TITLE
Issue #3679 Add test for controllers/casa_admins_controller.rb

### LINKED ISSUE #3679: Add test for controllers/casa_admins_controller.rb
Thank you for working on CASA!

To complete this issue:
1. open `controllers/casa_admins_controller.rb`
1. make a new test file: `spec/controllers/casa_admins_controller_spec.rb`
1. add at least one test for the functionality in controllers/casa_admins_controller.rb!
1. remove the line controllers/casa_admins_controller.rb from `.allow_skipping_tests`

This will improve our test coverage and make our code safer to modify and easier to understand.

Example:

```
Before:
require "rails_helper"

RSpec.describe CasaAdminsController do
  it "adds the numbers" do
    expect(described_class.new(1, 1)).to eq(2)
  end
end
```

### Questions? Join Slack!

We highly recommend that you join us in slack https://rubyforgood.herokuapp.com/ #casa channel to ask questions quickly and hear about office hours (currently Tuesday 6-8pm Pacific), stakeholder news, and upcoming new issues.

### PR DESCRIPTION
**Pull request by GALTdea**
### What github issue is this PR for, if any?
Resolves #3679
### What changed, and why?
Added casa_admins_controller_specs.rb test and added test for index action.

### How will this affect user permissions?
- Volunteer permissions:
- Supervisor permissions:
- Admin permissions:

### Feelings gif (optional)
What gif best describes your feeling working on this issue? https://giphy.com/
How to embed:
`alt text`

### Feedback please? (optional)
We are very interested in your feedback! Please give us some :) https://forms.gle/1D5ACNgTs2u9gSdh9

### CODE REVIEW

**Comment by mussajoop** [comment]
Hi, you might find it insightful to link this PR to an issue :)
`Resolves #2XXX` to `Resolves #3679`

**Comment by GALTdea** [comment]
> Hi, you might find it insightful to link this PR to an issue :) `Resolves #2XXX` to `Resolves #3679`

Thanks for the feedback. This is my first PR here. How long does it normally take them to respond?

**Comment by mussajoop** [comment]
> > Hi, you might find it insightful to link this PR to an issue :) `Resolves #2XXX` to `Resolves #3679`
> 
> Thanks for the feedback. This is my first PR here. How long does it normally take them to respond?

Not too long, I'd say people are quite dynamic here :). Since tonight there will be a working hour, I'd advise being at the meeting if you can, you'll surely have more info :)

**Comment by GALTdea** [comment]
> > > Hi, you might find it insightful to link this PR to an issue :) `Resolves #2XXX` to `Resolves #3679`
> > 
> > 
> > Thanks for the feedback. This is my first PR here. How long does it normally take them to respond?
> 
> Not too long, I'd say people are quite dynamic here :). Since tonight there will be a working hour, I'd advise being at the meeting if you can, you'll surely have more info :)

Thanks for the heads up! I'll probably be there.

File: app/controllers/case_court_reports_controller.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@ def show
       format.docx do
         @casa_case.latest_court_report.open do |file|
           # TODO test this .read being present, we've broken it twice now
```
Comment (compwron) [inline app/controllers/case_court_reports_controller.rb:22]: This TODO is now done :)

**Review by compwron — APPROVED** [review]
Nice :)

**Comment by GALTdea** [comment]
> Nice :)

Thank you!

**Pull request merged**